src/Response.php

```diff
@@ -194,9 +205,8 @@ public function getId()
      * @param  string $version
      * @return self
      */
-    public function setVersion($version)
+    public function setVersion(?string $version) : self
```

Comment:
Here we have inconsistent param type declaration with PHPDocs.
With LCS 2.0 we are not duplicating PHPDocs if nothing is changes. So this can be addressed later.

Reply:
Understood.